src/io/react-native/react-native-worker.ts

```diff
@@ -36,6 +36,9 @@ const hostnameBridgeProxyWhitelist = new Set<string>()
 // A map of domains that failed all CORS techniques and should not re-attempt CORS techniques
 const hostnameCorsProxyBlacklist = new Map<string, number>()
 
+// Throttle YAOB updates
+const YAOB_THROTTLE_MS = 75
```

Comment:
You also need to pass this to src/io/react-native/yaob-callbacks.ts, on line 80. That is where the bridge lives going the other way.

Since react-native code can't import web code or vice-versa, we need to put this constant in react-native-types.ts, which is "common ground" between the two sides.